docs/permissions/plugin-authors/03-adding-a-resource-permission-check.md

```diff
@@ -80,9 +82,10 @@ router.put('/todos', async (req, res) => {
   const decision = (
     await permissions.authorize(
       [{ permission: todoListUpdatePermission, resourceRef: req.body.id }],
-      {
-        token,
-      },
+      await auth.getPluginRequestToken({
+        onBehalfOf: credentials,
+        targetPluginId: 'permission',
+      }),
```

Comment:
credentials should be forwarded here instead, the permissions service no longer accepts a token